src/vue/common/TopBar.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="top-bar">
+    <div class="top-bar__main">
+      <slot name="main" />
+    </div>
+    <div class="top-bar__extra">
+      <slot name="extra" />
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
```

Comment:
let’s add a short usage manual. you can use `Drawer.vue` as a reference

Reply:
Resolved, added